openpype/hosts/houdini/api/plugin.py

```diff
@@ -256,8 +256,14 @@ def update_instances(self, update_list):
                 new_values,
                 update=True
             )
+
+            # Pop these keys as they error with setParms
+            new_values.pop("creator_attributes", None)
+            new_values.pop("publish_attributes", None)
+
             # Update values
-            instance_node.setParms(new_values)
+            if new_values:
+                instance_node.setParms(new_values)
```

Comment:
No no no no no no.

This is NOT the way.

`self.imprint` should be taking care or setting the new values - why do we need to explicitly call `setParms`?